modular_zubbers/code/modules/vending/lustwish.dm

```diff
@@ -18,6 +18,8 @@
 				/obj/item/clothing/neck/tie/bunnytie = 6,
 				/obj/item/clothing/under/costume/playbunny = 6,
 				/obj/item/clothing/suit/jacket/tailcoat = 6,
+				/obj/item/clothing/under/costume/loincloth/sensor = 6,
+				/obj/item/clothing/under/costume/loincloth/cloth/sensor = 6,
```

Comment:
Too ambuguous. It'll just be the same item. Just have the vendors dispense the sensor versions while the crafting recipe doesn't have sensors.

Reply:
But the vendors already dispense the sensor versions, and the crafting menu already crafts the no sensor versions?